### PR TITLE
Commiting Employee Testcase to Branch_X

### PR DESCRIPTION
**Pull request by Technoank**
Hi 
I have committed Test2 Employee Login. Please review it and approve it
Regards 
X

### CODE REVIEW

File: src/TestCases/EmployeeLogin.java [hunk 1]
```diff
@@ -0,0 +1,6 @@
+package TestCases;
+
+public class EmployeeloginvalidInput
+{
+
```
Comment (techieanki1247) [inline src/TestCases/EmployeeLogin.java:5]: Create a additional function if login fails

**Comment by techieanki1247** [comment]
Hi 
Your Code looks good . Please add one more function
Regards 
x

**Comment by Technoank** [comment]
Hi X,
I have added additional function . please Approve

**Comment by techieanki1247** [comment]
Hi X 
your code looks good . please go head with merge to master branch

**Pull request merged**